cartographer/io/probability_grid_points_processor.cc

```diff
@@ -80,13 +110,18 @@ ProbabilityGridPointsProcessor::FromDictionary(
                                   dictionary->GetBool("draw_trajectories"))
                                      ? DrawTrajectories::kYes
                                      : DrawTrajectories::kNo;
+  auto output_type = OutputType::kPng;
+  if (dictionary->HasKey("output_type")) {
```

Comment:
Is having a default value something we do in other places? Otherwise, why not simply change the configuration to contain this option?

Reply:
Yes, for instance draw_trajectories is true per default - just one line above.

Reply:
I think it makes sense to add a default here, so "old" configurations are still supported.